pkg/blockatlas/marketdata.go

```diff
@@ -15,7 +15,7 @@ type CoinType string
 
 type TickerResponse struct {
 	Currency string  `json:"currency"`
-	Result   Tickers `json:"result"`
+	Prices   Tickers `json:"prices"`
```

Comment:
should this be `docs` ? for consistency

Reply:
@lorenzozanotto ask me to change that to prices

Reply:
I think he meant `docs`

we should either keep `result` or `docs`

Reply:
@vikmeup @Pantani I asked to use `prices` because I took that from the `PaginationResponsePrices` struct used to draw the price charts from an array of prices. 

Using `docs` would be more consistent, I agree. Should we change also the field on the `/charts"` endpoint response in the TrustAPI as well? That's the only one call which uses `prices` instead of `docs` for a paginated response

Reply:
`/charts` will be re-build to blockatlas within the next 2-3 weeks, so we can adopt `docs` model as well.